### PR TITLE
Update remote work guidelines

### PR DESCRIPTION
**Pull request by EThom73**
**What:**
Update to remote working benefit

**Why:**
Updates remote working guidelines with some modifications:
- Language around wellbeing added
- Added amendments around ensuring team productivity not affected by remote working
- Guide to refusing requests for remote working for delivery leads
- Secruity guidelines for remote working

### CODE REVIEW

File: benefits/remote_working.md [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
-- Check that you're not needed on-site by a customer
-- Ensure you're available for a daily standup with your team
-- Be a good Slack citizen and check in with your team throughout your working day if possible
-- Ensure you're completely clear on your specific workload, down to the task level, for the day or days you wish to work from home, and ensure that you're certain you won't benefit from working closely with others on this
-- Ensure you're not scheduled for any meetings or other sessions
-- If you are in a public place, such as a coffee shop, make sure you are handling any sensitive data responsibly and securely. Can you be overseen? Is your connection secure? 
-- Barring any extraordinary circumstances, we like people to be in the office on Friday for [lunch](/benefits/friday_lunch.md) [(and hopefully drinks)](/benefits/friday_drinks.md)
+There are some additional considerations when working from home regularly
+
+- Talk to your team ahead of committing yourself to working remotely on a regular basis. Team members on Delivery should check in with the Delivery Lead. Members of business teams should check with their Head of Dept.
```
Comment (chrisblackburn) [inline benefits/remote_working.md]: does Dept. need to be abbreviated?

File: benefits/remote_working.md [hunk 2]
```diff
@@ -1,29 +1,41 @@
...
+
+### Refusing a Request for Remote Working
 
-## Regularly working from home
+All requests to work from home should be considered in their individual circumstances. The decision ultimately resides with teams, but below are some guidelines around when you may want to refuse a request to work from home
 
-There are additional considerations when regularly working remotely.
+- If the request will clash with and important ceremony with the client or the rest of the team (planning, retros, etc.)
+- If there are concerns the remote working will impact upon the productivity or success of the rest of the team
+- If there are concerns about productivity from a team member (this should be first addressed with the team member directly, and with their line manager if necessary
```
Comment (chrisblackburn) [inline benefits/remote_working.md]: should Line Manager be capitalised?

File: benefits/remote_working.md (hunk 1 above)
Comment (EThom73) [reply benefits/remote_working.md]: amended

File: benefits/remote_working.md (hunk 2 above)
Comment (EThom73) [reply benefits/remote_working.md]: amended

**Pull request merged**